src/summary/banner/patient-banner.component.tsx

```diff
@@ -17,7 +17,12 @@ export default function PatientBanner(props: PatientBannerProps) {
   return (
     <aside className={styles.patientBanner}>
       {!isLoadingPatient && !patientErr && (
-        <div className={styles.patientBanner}>
+        <div
+          className={styles.patientBanner}
+          role="button"
```

Comment:
Could you explain why you use the role property here? I'm not familiar with it.

Reply:
need the div element to behave like a button so as to allow the  toggle behavior when the user clicks anywhere on the div to expand the banner thus requiring to add the role attribute

Reply:
lgtm